2-functions/2-arguments/2-after.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+final class ReportGenerator
+{
+
+    // ...
+
+    public function generateSalesReport(User $user, DateRange $range)
+    {
+        $reportData = $this->repository->getSalesReportByUser($user->getId(), $range);
```

Comment:
```suggestion
        $reportData = $this->repository->findSalesReportByUser($user->getId(), $range);
```

I'd prefer `find...` because it's a call on repo